### ISSUE TITLE
/opt/chef/embedded/bin/inspec fails with Chef 15.3.14

### ISSUE THREAD

**Issue by mattray** [inspec/inspec#4511]
Testing with the latest stable release of Chef reports the following:
```
[root@install-chef-centos-7 bin]# /opt/chef/embedded/bin/inspec --version
Traceback (most recent call last):
    11: from /opt/chef/embedded/bin/inspec:23:in `<main>'
    10: from /opt/chef/embedded/bin/inspec:23:in `load'
     9: from /opt/chef/embedded/lib/ruby/gems/2.6.0/gems/inspec-core-bin-4.16.0/bin/inspec:10:in `<top (required)>'
     8: from /opt/chef/embedded/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
     7: from /opt/chef/embedded/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
     6: from /opt/chef/embedded/lib/ruby/gems/2.6.0/gems/inspec-core-4.16.0/lib/inspec/cli.rb:434:in `<top (required)>'
     5: from /opt/chef/embedded/lib/ruby/gems/2.6.0/gems/inspec-core-4.16.0/lib/inspec/cli.rb:434:in `new'
     4: from /opt/chef/embedded/lib/ruby/gems/2.6.0/gems/inspec-core-4.16.0/lib/inspec/plugin/v2/loader.rb:39:in `initialize'
     3: from /opt/chef/embedded/lib/ruby/gems/2.6.0/gems/inspec-core-4.16.0/lib/inspec/plugin/v2/loader.rb:277:in `detect_system_plugins'
     2: from /opt/chef/embedded/lib/ruby/2.6.0/rubygems/specification.rb:1033:in `find_by_name'
     1: from /opt/chef/embedded/lib/ruby/2.6.0/rubygems/dependency.rb:323:in `to_spec'
/opt/chef/embedded/lib/ruby/2.6.0/rubygems/dependency.rb:311:in `to_specs': Could not find 'inspec' (= 4.16.0) among 167 total gem(s) (Gem::MissingSpecError)
Checked in 'GEM_PATH=/root/.gem/ruby/2.6.0:/opt/chef/embedded/lib/ruby/gems/2.6.0', execute `gem env` for more information
    12: from /opt/chef/embedded/bin/inspec:23:in `<main>'
    11: from /opt/chef/embedded/bin/inspec:23:in `load'
    10: from /opt/chef/embedded/lib/ruby/gems/2.6.0/gems/inspec-core-bin-4.16.0/bin/inspec:10:in `<top (required)>'
     9: from /opt/chef/embedded/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:34:in `require'
     8: from /opt/chef/embedded/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:130:in `rescue in require'
     7: from /opt/chef/embedded/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:130:in `require'
     6: from /opt/chef/embedded/lib/ruby/gems/2.6.0/gems/inspec-core-4.16.0/lib/inspec/cli.rb:434:in `<top (required)>'
     5: from /opt/chef/embedded/lib/ruby/gems/2.6.0/gems/inspec-core-4.16.0/lib/inspec/cli.rb:434:in `new'
     4: from /opt/chef/embedded/lib/ruby/gems/2.6.0/gems/inspec-core-4.16.0/lib/inspec/plugin/v2/loader.rb:39:in `initialize'
     3: from /opt/chef/embedded/lib/ruby/gems/2.6.0/gems/inspec-core-4.16.0/lib/inspec/plugin/v2/loader.rb:277:in `detect_system_plugins'
     2: from /opt/chef/embedded/lib/ruby/2.6.0/rubygems/specification.rb:1033:in `find_by_name'
     1: from /opt/chef/embedded/lib/ruby/2.6.0/rubygems/dependency.rb:323:in `to_spec'
/opt/chef/embedded/lib/ruby/2.6.0/rubygems/dependency.rb:311:in `to_specs': Could not find 'inspec' (= 4.16.0) among 167 total gem(s) (Gem::MissingSpecError)
Checked in 'GEM_PATH=/root/.gem/ruby/2.6.0:/opt/chef/embedded/lib/ruby/gems/2.6.0', execute `gem env` for more information
[root@install-chef-centos-7 bin]# /opt/chef/embedded/bin/gem list inspec
*** LOCAL GEMS ***
debug_inspector (0.0.3)
inspec-core (4.16.0)
inspec-core-bin (4.16.0)
```

For reference, Chef 14.13 works fine:
```
[root@install-chef-centos-7 ~]# /opt/chef/embedded/bin/inspec --version
3.9.3
Your version of InSpec is out of date! The latest version is 4.16.0.
```

**Comment by clintoncwolfe** [comment]
Suspicion: when we scan for system plugins, we look for the `inspec` gemspec and list its deps; but the chef omnibus package does not include the `inspec` gem, instead the `inspec-core` gem.

Similar in spirit to #4431

**Comment by zenspider** [comment]
Fixed by #4513 which will go out today.
